core/views/ticket.py

```diff
@@ -1,6 +1,39 @@
 from django.shortcuts import render, redirect
+from django.http import JsonResponse
 from core.models import Ticket, Flight
-from django.db.models import F
+from label_printer.label_printer.boarding_pass_printer import generate_boarding_pass, print_boarding_pass
+from io import BytesIO
+from django.core.files import File
+from PIL import Image
+
+
+def generate_boarding_pass_for_ticket(ticket_id: int):
+    try:
+        ticket = Ticket.objects.get(id=ticket_id)
+    except Ticket.DoesNotExist:
+        raise ValueError(f"Ticket with ID {ticket_id} does not exist.")
+    image = generate_boarding_pass(
+                    ticket_id=ticket.id,
+                    lastname=ticket.costume,
+                    firstname=ticket.first_name,
+                    gate=ticket.flight.gate,
+                    departure_datetime=ticket.flight.scheduled_departure,
+                    boarding_datetime=ticket.flight.scheduled_departure,
+                    confirmation_number="AWGLUD",
```

Comment:
TODO: Generate random confirmation numbers that are unique to a particular flight